io/io/src/TBufferMerger.cxx

```diff
@@ -81,10 +80,7 @@ void TBufferMerger::WriteOutputFile()
             fQueue.pop();
          }
 
-         if (!buffer) {
-            done = true;
-            break;
-         }
+         if (!buffer) goto end;
```

Comment:
Because I need to break out of two loops. I could not find an easier way. The way I had before was using the `done` variable, but that gives us the deadlock...